packages/legend-graph/src/MetaModelConst.ts

```diff
@@ -401,3 +401,5 @@ export const DATE_TIME_FORMAT_WITH_MILLISECONDS =
   "yyyy-MM-dd'T'HH:mm:ss.SSSxxxx";
 export const DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ssxxxx";
 export const DATE_FORMAT = 'yyyy-MM-dd';
+
+export const DEPRECATED_STEREOTYPE = 'deprecated';
```

Comment:
Let's move it below `CORE_PURE_PATH` and rename this to `PURE_DEPRECATED_STEREOTYPE`